client/landing/subscriptions/components/add-sites-form/add-sites-form.tsx

```diff
@@ -56,6 +64,11 @@ const AddSitesForm = ( { onAddFinished }: AddSitesFormProps ) => {
 	);
 
 	const onAddSite = useCallback( () => {
+		if ( ! isLoggedIn ) {
+			setShowLoginDialog( true );
+			return;
+		}
```

Comment:
I wonder if we should move this to either:
* on updating the input - user starts typing and gets the dialog
* on clicking the 'add' button.

Currently, we show errors for incorrect URLs and only show the dialog modal if the user fully types in a working url to add. It seems like maybe a lot of work if they try to add something, get an error, meticulously type it or go find the copy to paste a second time, and then get interrupted with a signin flow:

![Screenshot 2025-02-14 at 7 04 46 AM](https://github.com/user-attachments/assets/7b65209e-16ff-4b67-baa9-d9f857681698)

^^ we may not want to let the user even get to that error point in the logged out flow.

Reply:
That makes sense. I'm inclined to leave this alone for now since we're [considering hiding this tab from logged-out users](https://github.com/Automattic/wp-calypso/pull/99745#discussion_r1956117789). I think we can leave it, and if we end up exposing this tab to logged-out users, revisit the issue.